test/dotnet-test.Tests/GivenDotnetTestBuildsAndRunsTestsWithDifferentOptions.cs

```diff
@@ -258,6 +258,88 @@ public void RunTestProjectSolutionWithArchOption_ShouldReturnZeroAsExitCode(stri
             result.ExitCode.Should().Be(ExitCodes.Success);
         }
 
+        [InlineData(TestingConstants.Debug)]
+        [InlineData(TestingConstants.Release)]
+        [Theory]
+        public void RunTestProjectSolutionWithRuntimeOption_ShouldReturnZeroAsExitCode(string configuration)
```

Comment:
I would maybe reword the last part as:

```suggestion
        public void RunTestProjectSolutionWithRuntimeOption_ShouldReturnExitCodeSuccess(string configuration)
```